### ISSUE TITLE
pre-defined cross validation datasets - custom training TF

### ISSUE THREAD

**Issue by moonlight2397** [PiotrekGa/pruned-cv#8]
I am doing Sentiment Analysis using neural networks with core Tensorflow. I am using the TPE sampler from Optuna with the Hyperband pruner, where the objective function minimizes the negative of the classification accuracy. The negative accuracy for each iteration is also reported for the Hyperband pruner to early stop trials. 

I want to combine the hyperparameter optimization with 10-fold cross-validation. I have 10 pre-defined dataset .txt files such that 'x' and 'y' cannot be defined directly as done below. Instead, I have a function that creates a feed_dict for each mini-batch where there are 6 'x' objects and 3 'y' objects in each feed_dict. 

`

data = fetch_california_housing()
x = data['data']
y = data['target']

`

For simple cross-validation, the model is run on each dataset in a loop, and the average accuracy over all 10 is calculated. How can I combine this with the hyperparameter optimization process with pruning using PrunedCV?

**Comment by PiotrekGa** [comment]
Hi, @moonlight2397! I see what you're trying to do. Unfortunately, this is not possible with the current state of the package. What you can do is to try to write a code that utilizes the general idea of printed cross-validation and suits your case. It's not a complicated concept after all.
